jwst/jump/tests/test_detect_jumps.py

```diff
@@ -28,13 +30,17 @@ def test_exec_time_0_crs(setup_inputs):
                                        nints=2, readnoise=6.5, gain=5.5,
                                        grouptime=2.775, deltatime=2.775)
 
-    tstart = time.time()
+    tstart = time.monotonic()
     # using dummy variable in next to prevent "F841-variable is assigned to but never used"
     _ = run_detect_jumps(model, gain, rnoise, 4.0, 5.0, 6.0, 'none', 200, 4, True)
-    tstop = time.time()
+    tstop = time.monotonic()
 
     t_elapsed = tstop - tstart
-    MAX_TIME = 10  # takes 1.6 sec on my Mac
+    # sometimes this takes a long time on github mac osx runners
+    if platform.system() == 'Darwin' and os.environ.get('CI', False):
+        MAX_TIME = 60
```

Comment:
Given that the original code mentions a time of 1.6 sec on a personal mac, a new limit of 60 sec seems unusually high to me. That's a factor of nearly 40x and just the kind of increase this test is supposed to be looking for. Could we maybe at least downgrade the new limit to something like 20?  Even that is a large increase relative to the original 1.6.

Reply:
Thanks for looking at this. I switched the `MAX_TIME` to 20 for the mac github runners. The linux runners still use the same 10 seconds as before (as do any local tests or jenkins linux runs). I'm not certain the jenkins mac runs set the CI variable or not but can look into that if there's a concern for relaxing the threshold for those runs.

I also updated the comment to point to a closed (but still active) issue for slow/inconsistent mac runners which illustrates that this is a general issue and nothing specific to jwst:
https://github.com/actions/runner-images/issues/1336